atlite/convert.py

```diff
@@ -501,7 +500,10 @@ def wind(cutout, turbine, smooth=False, **params):
         A turbineconfig dictionary with the keys 'hub_height' for the
         hub height and 'V', 'POW' defining the power curve.
         Alternatively a str refering to a local or remote turbine configuration
-        as accepted by atlite.resource.get_windturbineconfig().
+        as accepted by atlite.resource.get_windturbineconfig(). Locally stored turbine
+        configurations can also be modified with this function. E.g. to setup a different hub
+        height from the one used in the yaml file,one would write
+                "turbine=get_windturbineconfig(“NREL_ReferenceTurbine_5MW_offshore”)|{“hub_height”:120}"
```

Comment:
I am wondering where this change is coming from.

Reply:
Good change!

Reply:
@FabianHofmann just a small doc addition explaining how to modify the wind hub height. This is unrelated to the pv tracking update